import java.util.Scanner;
public class CalculatorSwitchCase {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double first = input.nextDouble();
        double second = input.nextDouble();
        String op = input.next();
        switch(op) {
            case "+":
                System.out.println(first + second);
                break;
            case "-":
                System.out.println(first - second);
                break;
            case "*":
                System.out.println(first * second);
                break;
            case "/":
                if(second != 0)
                    System.out.println(first / second);
                else
                    System.out.println("Division by zero error");
                break;
            default:
                System.out.println("Invalid Operator");
                break;
        }
        input.close();
    }
}